test/k8s-integration/main.go

```diff
@@ -298,27 +300,52 @@ func handle() error {
 			}
 		}()
 	}
+
 	// For windows cluster, when cluster is up, all Windows nodes are tainted with NoSchedule to avoid linux pods
 	// being scheduled to Windows nodes. When running windows tests, we need to remove the taint.
 	if *platform == "windows" {
 		nodesCmd := exec.Command("kubectl", "get", "nodes", "-l", "beta.kubernetes.io/os=windows", "-o", "name")
 		out, err := nodesCmd.CombinedOutput()
 		if err != nil {
-			return fmt.Errorf("failed to get nodes: %v", err)
+			return fmt.Errorf("failed to get windows nodes: %v", err)
 		}
 		nodes := strings.Fields(string(out))
 		for _, node := range nodes {
 			taintCmd := exec.Command("kubectl", "taint", "node", node, "node.kubernetes.io/os:NoSchedule-")
-			_, err = taintCmd.CombinedOutput()
+			out, err := taintCmd.CombinedOutput()
 			if err != nil {
 				return fmt.Errorf("failed to untaint windows node %v", err)
 			}
+			klog.Infof("untaint windows nodes: %s, output %s", node, string(out))
+		}
+		/*nodesCmd = exec.Command("kubectl", "get", "nodes", "-l", "beta.kubernetes.io/os=linux", "-o", "name")
```

Comment:
can we remove this code?

Reply:
oh, sorry. I forgot to push my latest update there.